docs/jazzy.yml

```
@@ -181,7 +181,7 @@ custom_categories:
       - RoadObjectMatcher
       - RoadObjectsStore
       - RoadObjectLocation
-      - RoadObjectType
+      - ObjectType
```

This needs to be fully qualified in order for jazzy to know what it refers to.

```suggestion
      - RoadObject.Kind
```